POEApi.Model/ProxyMapper.cs

```diff
@@ -298,5 +299,18 @@ internal static List<Tab> GetTabs(List<JSONProxy.Tab> tabs)
                 throw;
             }
         }
+
+        public static ChargeInfo GetCharges(List<JSONProxy.Property> list)
+        {
+            var chargeSize = list.Find(p => p.Name == CHARGES);
+            if (chargeSize == null)
+                return new ChargeInfo(0, 0);
+
+            var qty = (object[]) chargeSize.Values[0];
```

Comment:
That is rather odd, but then that's the GGG api for you ;)

Thanks for looking into the json, looks like we'll have to stick with this for now!